src/test/descriptor_tests.cpp

```diff
@@ -100,9 +103,13 @@ void Check(const std::string& prv, const std::string& pub, int flags, const std:
                     BOOST_CHECK_MESSAGE(SignSignature(Merge(keys_priv, script_provider), spks[n], spend, 0, 1, SIGHASH_ALL), prv);
                 }
             }
```

Comment:
Maybe add a comment:
`// Check if origin information (master key fingerprint, derivation path) is present`

It's actually not obvious to me how this test checks that an origin string passed into the test is actually stored correctly.

Reply:
Added some comments.